OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -32,6 +32,14 @@ public enum DamageState
 		Dead = 32
 	}
 
+	public enum BlockedByActor
```

Comment:
Done.

As an aside, I was under the impression that this is actually the most common usage of enums. It is literally the first example in https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/enum . The way it is used throughout the OpenRA code-base, as a way to fit multiple booleans in a byte, seems more unusual to me. 

It is also not always justified. just take the `DamageState` enum just above this as an example. Those categories are clearly mutually exclusive, so there is no reason for them to occupy orthogonal bits. They could just be 0 to 5.

Reply:
By ordering, I meant the fact your are comparing enum values with `<` or `>` operators instead of just direct equality. Nothing wrong with it - but not that common. Bitwise packing as usually indicated by the `Flags` attribute is something else.

Usually, if I happen across `enum Foo { A, B, C }` I don't really expect consuming code to break if I reorder that to `enum Foo { A, C, B }`. That's what I wanted the comment to guard against (e.g. stop passers by reordering it alphabetically or sticking a new value in "the wrong place", or whatever)

I agree with you r.e. `DamageState` - odd that it's decided for `Flags` there.